packages/curve-ui-kit/src/features/connect-wallet/ui/ConnectWalletPrompt.tsx

```diff
@@ -16,16 +24,62 @@ export const ConnectWalletPrompt = ({
   connectWallet,
   isLoading,
 }: ConnectWalletPromptProps) => (
-  <ConnectWallet
-    connectWallet={connectWallet}
-    description={description}
-    connectText={connectText}
-    loadingText={loadingText}
-    isLoading={isLoading}
-    theme={useUserProfileStore((state) => state.theme)}
-  />
+  <Stack
+    padding={7}
+    sx={{
+      backgroundColor: 'var(--table--background-color)',
+      maxWidth: '50rem',
+    }}
+  >
+    <Box display="flex" position="relative" width="100%">
+      <Box
```

Comment:
I think we can get rid of this Box and just use a (non-absolute-positioned) Stack with
```
  sx={{
    backgroundImage: `url(${getBackgroundUrl(useUserProfileStore((state) => state.theme))})`,
    backgroundSize: 'contain',
    backgroundPosition: 'center',
    backgroundRepeat: 'no-repeat',
  }}
```